jobs/ci-run/integration/common/test-runner.sh

```diff
@@ -32,6 +32,23 @@ while sudo lsof /var/lib/apt/lists/lock 2> /dev/null; do
 done
 sudo apt-get -y update
 
+setup_parca_agent() {
+    # Parca agent is already installed, then skip it.
+    if [ "$(which parca-agent >/dev/null 2>&1)" ]; then
+        return
+    fi
+
+    sudo snap install parca-agent --classic
+    sudo snap set parca-agent metadata-external-labels="machine=ci-run-${{BOOTSTRAP_PROVIDER}}"
+
+    # Hide the setting of the bearer token to the parca agent.
+    set +x
+    {
+        sudo snap set parca-agent remote-store-bearer-token="${{PARCA_BEARER_TOKEN}}"
```

Comment:
The `$PARCA_BEARER_TOKEN` is setup in the credentials.

Reply:
Do you need to set up an ` || true` in case it fails? I also don't know if this sort of thing ends up in bash history, etc.